README.md

```diff
@@ -45,11 +45,13 @@ Configure the library in `config/config.exs`:
 ```elixir
 config :avrora,
   registry_url: "http://localhost:8081",
+  registry_auth: {:basic, "username", "password"}
   schemas_path: Path.expand("./priv/schemas"),
   names_cache_ttl: :timer.minutes(5)
 ```
 
 - `registry_url` - URL for the Confluent Schema Registry, default `nil`
+- `registry_auth` - Authentication for the Confluent Schema Registry, default `nil`
```

Comment:
I've checked other auth methods on Schema Registry and I would say we have to stick to a simple once first. Also, an interface for more complex will not look similar.

So first of all, I would suggest a very simple interface which will not evolve to anything else:

```elixir
registry_basic_auth: ["username", "password"]
```

Second, this library does as much as it can on compile-time and rest as cached settings. But `file` reading will happen every time and it's very costly. Also if someone can put a file, then it's definitely possible to set an environment variable. Hence, I would suggest to scope file reading.